### DISCUSSION TITLE
Stuck on login screen, can't remember username

### DISCUSSION THREAD

**Discussion by risbi0** [simeononsecurity/Windows-Optimize-Harden-Debloat#76] [Q&A] [answered]
My login now asks for a username when I restarted my pc to apply the changes. The problem is I don't remember my username at all. Is there a way of getting it without logging in, or maybe bypassisg it temporarily? I do have computers that share the same network as mine maybe that can help.

**Comment by risbi0** [comment]
To add, I haven't changed the username since running the script, and I think the script changed the username to a default one and I haven't checked it.

**Reply by simeononsecurity** [reply]
the only time it changes user names is with the built into windows accounts. Accounts you'd only be using on windows server, enterprise, and potentially IOT and embedded editions.  If you're running Home or Pro, you'd have to specifically try to do that. So I doubt that is the issue.

**Comment by risbi0** [comment, marked as answer]
Hello again. I finally remembered my username and I logged in now. Sorry for the bother.
